astro/package.json

```diff
@@ -19,11 +19,11 @@
 		"@astrojs/tailwind": "^5.0.2",
 		"@docsearch/js": "^3.3.3",
 		"@docsearch/react": "^3.3.3",
+		"@fusionauth/astro-components": "^1.0.0",
```

Comment:
do we want to be on 1.1.0 now that it is officially released?

Reply:
It was already on 1.1.0 on `package-lock.json` but I bumped it here as well. Thanks!